worlds/noita/Regions.py

```diff
@@ -15,27 +15,30 @@ def create_region(world: MultiWorld, player: int, region_name: str) -> Region:
         location = Locations.NoitaLocation(player, location_name, location_data.id, new_region)
         opt_orbs = world.orbs_as_checks[player].value
         opt_bosses = world.bosses_as_checks[player].value
+        opt_paths = world.path_option[player].value
+        opt_hc = world.hidden_chests[player].value
+        # opt_peds = world.pedestal_checks[player].value
         ltype = location_data.ltype
         flag = location_data.flag
+
         if flag == 0 or ltype == "orb" and flag <= opt_orbs or ltype == "boss" and flag <= opt_bosses:
             new_region.locations.append(location)
 
-    return new_region
-
-
-# Creates chest locations in the Forest region (HACK), making sure not to generate more locations than requested
-def create_chests(world: MultiWorld, player: int, num_locations: int) -> Region:
-    new_region = Region("Forest", player, world)
-    total_locations = world.total_locations[player].value - num_locations
-
-    for i in range(total_locations):
-        location_name = f"Chest{i+1}"
-        location_id = 110000+i
-
-        location = Locations.NoitaLocation(player, location_name, location_id, new_region)
-        location.progress_type = LocationProgressType.DEFAULT
-
-        new_region.locations.append(location)
+        if ltype == "hc" and flag <= opt_paths:
+            for i in range(opt_hc):
+                location_name_num = f"{location_name} {i+1}"
+                location_id = location_data.id + i
+                location = Locations.NoitaLocation(player, location_name_num, location_id, new_region)
+                location.progress_type = LocationProgressType.DEFAULT
+                new_region.locations.append(location)
+
+        # if ltype == "peds" and flag <= opt_paths:

```

Comment:
I'm guessing this is commented out until pedestals are implemented?

Reply:
That's correct, just wrote it in advance